dashboard/public/pixelation/pixelation.html

```diff
@@ -90,8 +90,42 @@
     $('#below_viz_instructions').text(options.instructions).show();
   }
   pixelationInit();
-  pixel_data.value = options.data;
-  drawGraph();
-  if(options.format=='raw') unformatBits()
-  else formatBitDisplay();
+
+  function pixelationDisplay() {
+    pixel_data.value = options.data;
+    drawGraph();
+    formatBitDisplay();
+  }
+
+  // Initialize projects for save/load functionality if channel id is present.
+  if (appOptions.channel) {
+    window.apps.setupProjectsExternal();
+    var projectsHandler = {
+      setInitialLevelHtml: function (levelHtml) {},
+      getLevelHtml: function () {
+        return '';
+      },
+      setInitialLevelSource: function (levelSource) {
+        options.data = levelSource;
```

Comment:
One thing we could do, that is probably out of the scope of this PR, would be to have pixelation.js export a single object or constructor (where exporting means attaching to the global namespace). We could then do something like var pixelation = new Pixelation(options), and call methods on pixelation.